apis/cc-de_CH/atrium-api-cc-de_CH-jvm/src/test/kotlin/ch/tutteli/atrium/api/cc/de_CH/IterableContainsInOrderOnlyEntriesAssertionsSpec.kt

```diff
@@ -20,6 +21,14 @@ class IterableContainsInOrderOnlyEntriesAssertionsSpec : Spek({
         "[Atrium][Builder] "
     )
 
+    object OldShortcutSpec : ch.tutteli.atrium.spec.integration.IterableContainsInOrderOnlyEntriesAssertionsSpec(
```

Comment:
Good idea, but that's actually covered by backward compatibility tests (Tests from 0.7.0 run against 0.8.0 -> see jobs bc (source backward compatibility) and bbc (binary backward compatibility) on travis. I should have mentioned it, sorry for that.

Reply:
Done.

Reply:
It's still here :) did you push the wrong commit?

Reply:
Sorry, wanted to rebase locally before pushing. Anyways it's pushed now and it should be okay :) 

I'll go through all the changes just for a second check, and if it seems okay I'll remove the WIP status.

Reply:
The build will still fail for the same reason as mentioned above. Let me know if you are fed up with fixing, I can also do this last bit (kind of a review as well).